### PR TITLE
Rearchitect the docs navigation

### PR DESCRIPTION
**Pull request by dwightjl**
- Reorganize the top navigation by service and technology
- Move Solana into Data Feeds next to Starknet
- Merge Solana Data Feeds lists with the EVM contract addresses lists
- Enable the Resources section to be global without switching the navigation
- Update video tutorial URLs on the landing page
- Add Resources and Architecture to several sections as global sections
- Clean up homepage product card code

### CODE REVIEW

File: src/features/redirects/redirects.json [hunk 1]
```diff
@@ -397,17 +397,17 @@
...
-      "destination": "solana/data-feeds/using-data-feeds-solana",
+      "destination": "data-feeds/solana/using-data-feeds-solana",
       "statusCode": 301
     },
     {
       "source": "solana/using-data-feeds-off-chain",
-      "destination": "solana/data-feeds/using-data-feeds-off-chain",
+      "destination": "data-feeds/solana/using-data-feeds-off-chain",
       "statusCode": 301
     },
```
Comment (thedriftofwords) [inline src/features/redirects/redirects.json:412]: Don't forget to redirect the base /solana and /ethereum pages

**Review by thedriftofwords — DISMISSED** [review]

File: src/features/redirects/redirects.json (hunk 1 above)
Comment (dwightjl) [reply src/features/redirects/redirects.json:412]: Done

File: src/features/feeds/components/FeedList.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const FeedList = ({
   ecosystem?: string
   initialCache?: Record<string, ChainMetadata>
 }) => {
-  const chains = ecosystem === "deprecating" ? ALL_CHAINS : ecosystem === "solana" ? SOLANA_CHAINS : CHAINS
+  const chains = CHAINS
```
Comment (fmonper1) [inline src/features/feeds/components/FeedList.tsx:24]: why dont we just use the `CHAINS` object?

File: src/features/landing/components/Tabs.tsx [hunk 3]
```diff
@@ -1,32 +1,15 @@
...
 import tabs from "./Tabs.module.css"
 import { ProductCard } from "./ProductCard"
-import solanaIcon from "../assets/solana-chains.svg"
-import ethereumIcon from "../assets/evm-chains.svg"
-import { evmProducts, solanaProducts } from "../data"
+import { evmProducts } from "../data"
 
 export const Tabs = () => {
-  const [tabId, setTabId] = useState("evm-products")
+  const [tabId] = useState("evm-products")
```
Comment (fmonper1) [inline src/features/landing/components/Tabs.tsx:9]: If we don't have the tabs we can remove the useState logic

File: src/features/feeds/components/FeedList.tsx (hunk 2 above)
Comment (dwightjl) [reply src/features/feeds/components/FeedList.tsx:24]: Yes this should work now that chains.ts is simplified. I'll change it.

File: src/features/landing/components/Tabs.tsx (hunk 3 above)
Comment (dwightjl) [reply src/features/landing/components/Tabs.tsx:9]: I was thinking I should leave it for the future if we need the extra capability, but we can always come back and get it again later. I'll update this.

File: src/features/feeds/components/FeedList.tsx (hunk 2 above)
Comment (dwightjl) [reply src/features/feeds/components/FeedList.tsx]: Looks like we still need this for the feed deprecation lists. I restored part of it.

File: src/features/landing/components/Tabs.tsx (hunk 3 above)
Comment (dwightjl) [reply src/features/landing/components/Tabs.tsx]: Removed

**Review by ritika-b — APPROVED** [review]

**Comment by dwightjl** [comment]
Manually rebased to resolve GitHub's confusion

**Pull request merged**